.vscode/settings.json

```diff
@@ -5,5 +5,6 @@
   ],
   "phpcs.composerJsonPath": "src",
   "php.suggest.basic": false,
-  "git.ignoreLimitWarning": true
+  "git.ignoreLimitWarning": true,
+  "terminal.integrated.cwd": "${workspaceFolder}/docker"
```

Comment:
This seems like a personal preference, and not something you are intending to push to all developers, so I would leave this out.
```suggestion
```